### PR TITLE
Origin/add/shuffleboard selector for auto nav challenge

### PR DESCRIPTION
**Pull request by NolanCampbell2357**
Also creates flag to handle whether robot should run code specific to in-person or at-home challenges.

### CODE REVIEW

File: .wpilib/wpilib_preferences.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "currentLanguage": "none",
+    "enableCppIntellisense": false,
+    "projectYear": "none",
```
Comment (Stick-2357) [inline .wpilib/wpilib_preferences.json]: Change to 2021

File: robotbase/src/main/java/com/systemmeltdown/robot/commands/TemporaryCommand.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
+// Copyright (c) FIRST and other WPILib contributors.
```
Comment (Stick-2357) [inline robotbase/src/main/java/com/systemmeltdown/robot/commands/TemporaryCommand.java]: Dead file

File: robotbase/src/main/java/frc/robot/RobotContainer.java [hunk 3]
```diff
@@ -105,6 +109,8 @@ public RobotContainer() {
       m_shootSub,
       m_visionSub
     );
+
```
Comment (Stick-2357) [inline robotbase/src/main/java/frc/robot/RobotContainer.java]: Feature Flag for initialization needed

**Review by Stick-2357 — CHANGES_REQUESTED** [review]

File: .wpilib/wpilib_preferences.json (hunk 1 above)
Comment (NolanCampbell2357) [reply .wpilib/wpilib_preferences.json]: Done.

File: robotbase/src/main/java/frc/robot/RobotContainer.java (hunk 3 above)
Comment (NolanCampbell2357) [reply robotbase/src/main/java/frc/robot/RobotContainer.java]: Done

File: robotbase/src/main/java/com/systemmeltdown/robot/commands/TemporaryCommand.java (hunk 2 above)
Comment (NolanCampbell2357) [reply robotbase/src/main/java/com/systemmeltdown/robot/commands/TemporaryCommand.java]: Done.

**Review by Stick-2357 — APPROVED** [review]

**Pull request merged**